lib/logstash-docket/util/alias_plugin_util.rb

```diff
@@ -0,0 +1,20 @@
+# encoding: utf-8
+
+require "yaml"
+require "net/http"
+
+module LogstashDocket
+  module Util
+    ##
+    # A util class defines repetitive logics for aliased plugins.
+    #
+    class AliasPluginUtil
```

Comment:
Nitpick: The name `AliasPluginUtil` inside a `Util` namespace is a bit redundant and not very descriptive.

I would suggest `Util::AliasMappings` instead.

Reply:
Thanks for the tip. Honestly, there were several namings pointed same thing. I have make it unique saying `alias definitions`. So form this perspective naming the util as `AliasDefinitionsLoader` looked good to me. Let me know your thoughts.